android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java

```diff
@@ -197,17 +197,17 @@ private void add(TiUIView child, int childIndex)
 							((ViewGroup) nv).addView(cv, child.getLayoutParams());
 						}
 					}
-					if (children.contains(child)) {
-						children.remove(child);
-					}
-					if (childIndex == -1) {
-						children.add(child);
-					} else {
-						children.add(childIndex, child);
-					}
-					child.parent = proxy;
 				}
 			}
+			if (children.contains(child)) {
```

Comment:
Move this into outer conditional block, as it would not add the child if its `nativeView` was `null`